crates/proto/proto/responses.proto

```diff
@@ -45,11 +45,15 @@ message SyncStateResponse {
     // List of account hashes updated after `block_num + 1` but not after `block_header.block_num`
     repeated account.AccountSummary accounts = 5;
 
+    // List of transaction IDs which changed the state of requested accounts
+    // between `block_num + 1` and `block_header.block_num`
+    repeated digest.Digest transactions = 6;
```

Comment:
I would update the comment to be: "List of IDs for transactions executed against requested accounts between `block_num + 1` and `block_header.block_num`."